src/release-it-config/README.md

```diff
@@ -0,0 +1,100 @@
+# Silvermine-release
```

Comment:
Also, running `silvermine-release` with no arguments does nothing. Is it possible to default it to the "help" command?

Reply:
Is there a way to configure a custom CHANGELOG header / footer? Not a deal-breaker, but just curious since in some of our CHANGELOGs we've had this as the header:

```md
# Changelog

All notable changes to this project will be documented in this file.
See [our coding standards][commit-messages] for commit guidelines.
```

and this as the footer:

```md
[commit-messages]: https://github.com/silvermine/silvermine-info/blob/master/commit-history.md#commit-messages
```

Reply:
Updated the CLI tool to default to the help command.

As regards the other points:

 **release:tag warning**

This is release-it's default behavior, and [doesn't appear to be configurable](https://github.com/release-it/release-it/blob/master/lib/plugin/git/Git.js#L142).

However, it could be made so...

**changelog chore output in console**

This is conventional-changelog's default behavior, and I don't immediately see a way to override it. CC will report any commits in the range we provide it, so it seems like with a little `git describe` filtering or similar we could filter them out.

**custom changelog header**

This appears to be functionality specific to standard-version, would be nice to have, but it doesn't appear to be possible with release-it or conventional-changelog, so we'd have to roll our own.

Reply:
opened #7 and #8

Reply:
Also #9